manifests/compute/ec2.pp

```diff
@@ -5,9 +5,8 @@
 # === Parameters:
 #
 # $version::  Package version to install, defaults to installed
-#             type:Optional[String]
 #
-class foreman::compute::ec2($version = 'installed') {
+class foreman::compute::ec2(Optional[String] $version = 'installed') {
```

Comment:
No, it wasn't really - I'll revert it back to `String`. There was a bug (http://projects.theforeman.org/issues/6158) in Kafo which was why this was made `Optional` in 3f5238c.